pyQuARC/schemas/check_messages.json

```diff
@@ -183,6 +183,22 @@
         },
         "remediation": "The temporal Beginning Date Time and Ending Date Time are identical. Recommend removing the Beginning Date Time and Ending Date Time elements and instead provide a Single Date Time."
     },
+    "project_date_time_logic_check_1": {
+        "failure": "The Start Date Time `{}` comes after the End Date Time `{}`.",
+        "help": {
+            "message": "",
+            "url": ""
+        },
+        "remediation": "Update the start and/or end date time so that the timeline is logical."
+    },
+    "project_date_time_logic_check_2": {
+        "failure": "The Start Date Time `{}` and End Date Time `{}` are the same.",
+        "help": {
+            "message": "",
+            "url": ""
+        },
+        "remediation": "The temporal Start Date Time and End Date Time are identical."
+    },
```

Comment:
Why does one remediate refer to the temporal dates but the other just refers to regular dates? Can we standardize the use of capitalization between the two messages?

Reply:
fixed in 72f7f19